### DISCUSSION TITLE
Analyzer warning when using Unicode Tags.

### DISCUSSION THREAD

**Discussion by StevenTCramer** [statiqdev/Discussions#169] [Q&A] [answered]
I have added a tag for posts in the Thai language.

When I use `Thai` all is good but when I use `ไทย` then the site works perfectly fine, but the analyzer gives me:

```console
...
 Could not validate relative link: <a href="/tags/&amp;#xE44;&amp;#xE17;&amp;#xE22;" 
...
```

I'm not sure how it is trying to do the analysis of the link.

**Comment by daveaglick** [comment, marked as answer]
This is definitely a bug, going to convert it to an issue for tracking.

https://github.com/statiqdev/Statiq.Web/issues/997

**Reply by daveaglick** [reply]
This was indeed a bug and has been resolved - fix will go out with Statiq Web beta 50 (can't believe we're up to 50 betas already, going to need to roll that over to final at some point I guess).
